### PR TITLE
Feature/card

### PR DESCRIPTION
**Pull request by zimb0o**
I have created a simple card component that displays information about different spots. The card includes the name of the spot, its address, and a picture.

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -1,7 +1,18 @@
...
+import React from "react";
+import styled from "styled-components";
+import SpotCard from "../components/SpotCard.js";
+
+export default function App() {
   return (
     <div>
-      <h1>Hello from Next.js</h1>
+      <h1>TheCaliSpots</h1>
+      <SpotCard name={"Alex"} adress={"asdfsgsd"} />
```
Comment (ThomasKirst) [inline pages/index.js:9]:
```suggestion
      <SpotCard name={"Alex"} address={"asdfsgsd"} />
```

File: pages/index.js [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
     </div>
   );
 }
+
+const headline = styled.h1`
+  margin: 10px;
+  padding: 20px;
+  left: 30px;
+`;
```
Comment (ThomasKirst) [inline pages/index.js:18]: This is not used here, right?

File: components/SpotCard.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+            src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/8e/Hauskatze_langhaar.jpg/600px-Hauskatze_langhaar.jpg"
+            alt="Spot Image"
+          />
+        </ImageContainer>
+      </Content>
+    </Card>
+  );
+}
+
+const Card = styled.div`
```
Comment (ThomasKirst) [inline components/SpotCard.js:22]: You could consider making this an `article` or `section` tag instead of a `div`.

File: components/Header.js
Comment (ThomasKirst) [inline components/Header.js:1]: I don´t see this file used anywhere. You can delete it.

**Review by ThomasKirst — CHANGES_REQUESTED** [review]
Great start so far 👍 
I found a little misspelling issue and an unused component.
Apart from that, very good first PR :)

File: pages/index.js (hunk 2 above)
Comment (zimb0o) [reply pages/index.js:18]: Yes, I tried something and honestly didn't stick to the US guidelines. I will change that in the upcoming PR. The same goes for the third point. I will make sure to pay attention to it.

File: components/SpotCard.js (hunk 3 above)
Comment (zimb0o) [reply components/SpotCard.js:22]: Thank you!

File: pages/index.js (hunk 2 above)
Comment (zimb0o) [reply pages/index.js:18]: Yes, I tried something and honestly didn't stick to the US guidelines. I will change that in the upcoming PR. The same goes for the third point. I will make sure to pay attention to it.

**Pull request merged**